### PR TITLE
Condense Weapon Expertise class feature

### PR DESCRIPTION
**Pull request by ammalagonc**
Re-do of #16455
As it turns out, deleting the class features was creating a cascade effect with actors from the Kingmaker compendium. Did not want to touch those yet without a discussion.

### CODE REVIEW

File: packs/classfeatures/weapon-expertise.json [hunk 1]
```diff
@@ -32,13 +32,79 @@
...
+                ]
+            },
+            {
+                "itemId": "{item|id}",
+                "key": "ItemAlteration",
+                "mode": "override",
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.ClassFeatures.WeaponExpertise.{actor|class.slug}.Title"
```
Comment (TikaelSol) [inline packs/classfeatures/weapon-expertise.json:95]:
```suggestion
                        "text": "PF2E.SpecificRule.ClassFeatures.WeaponExpertise.Title"
```
I actually don't think we need this at all. the ones that have different names shouldn't get condensed.

File: static/lang/re-en.json [hunk 2]
```diff
@@ -2457,6 +2457,57 @@
...
                         "SpellSwapping": "As you gain new spells in your repertoire, you might want to replace some of the spells you previously learned. Each time you gain a level and learn new spells, you can swap out one of your old spells for a different spell of the same level. If it's a level at which you lose a set of lower-level slots, you can replace the two in either order. You can also instead swap a cantrip. You can also swap out spells by retraining during downtime."
                     }
+                },
+                "WeaponExpertise": {
+                    "alchemist": {
+                        "Description": "You've trained to more effectively wield the weapons you find in your lab. Your proficiency ranks for simple weapons, alchemical bombs, and unarmed attacks increase to expert."
+                    },
```
Comment (TikaelSol) [inline static/lang/re-en.json:2464]: We don't need these notes for classes that didn't have their feature condensed.

File: packs/classes/alchemist.json [hunk 3]
```diff
@@ -111,6 +111,12 @@
                 "name": "Medium Armor Expertise",
                 "uuid": "Compendium.pf2e.classfeatures.Item.Medium Armor Expertise"
             },
+            "ca55v": {
+                "img": "icons/skills/melee/hand-grip-sword-red.webp",
+                "level": 7,
+                "name": "Weapon Expertise",
+                "uuid": "Compendium.pf2e.classfeatures.Item.Weapon Expertise"
+            },
```
Comment (TikaelSol) [inline packs/classes/alchemist.json:119]: Make sure to revert the ones that are staying

Comment (ammalagonc) [reply packs/classes/alchemist.json]: Files refreshed - please check when you get a moment.

File: packs/classfeatures/weapon-expertise.json [hunk 4]
```diff
@@ -39,6 +39,27 @@
...
+                "mode": "override",
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.ClassFeatures.WeaponExpertise.Title"
+                    },
+                    {
+                        "text": "PF2E.SpecificRule.ClassFeatures.WeaponExpertise.{actor|class.slug}.Description"
+                    }
+                ]
```
Comment (TikaelSol) [inline packs/classfeatures/weapon-expertise.json:62]:
```suggestion
                "value": [
                    {
                        "text": "PF2E.SpecificRule.ClassFeatures.WeaponExpertise.{actor|class.slug}.Description"
                    }
                ]
```

File: static/lang/re-en.json [hunk 5]
```diff
@@ -2457,6 +2457,36 @@
...
+                        "Description": "You've dedicated yourself to learning the intricacies of your weapons. Your proficiency ranks for simple weapons, martial weapons, and unarmed attacks increase to expert."
+                    },
+                    "oracle": {
+                        "Description": "You've dedicated yourself to learning the intricacies of your weapons. Your proficiency ranks for simple weapons, martial weapons, and unarmed attacks increase to expert."
+                    },
+                    "sorcerer": {
+                        "Description": "You've improved your combat skill. Your proficiency ranks for simple weapons and unarmed attacks increase to expert."
+                    },
+                    "Title": "**Weapon Expertise**",
+                    "witch": {
```
Comment (TikaelSol) [inline static/lang/re-en.json:2484]:
```suggestion
                    },
                    "witch": {
```

**Pull request merged**